test/e2e/validation_test.go

```diff
@@ -43,6 +45,35 @@ func testWorkerLabel(t *testing.T) {
 	}
 }
 
+// wmcoVersion returns the version of the operator. This is sourced from the WMCO binary used to create the operator image.
+// This function will return an error if the binary is missing.
+func wmcoVersion() (string, error) {
```

Comment:
Given this is not associated with a struct, getWMCOVersion() would be more appropriate.